shared/test/test_assets.rb

```diff
@@ -3,6 +3,7 @@
 require File.expand_path '../../../deployment', __FILE__
 require File.expand_path '../../middleware/files_api', __FILE__
 require File.expand_path '../../middleware/channels_api', __FILE__
+require File.expand_path '../../middleware/helpers/asset_bucket', __FILE__
```

Comment:
Hmm we're getting a mix of API and model tests here.  I wonder if it would be good to separate them?  I.e. test `AssetBucket` and `FilesApi` in separate files.